cert-manager-configs/templates/Cluster.yml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.cluster.apiServer.enabled }}
+{{- if .Values.cluster.ingressController.enabled }}
+---
+kind: APIServer
+apiVersion: config.openshift.io/v1
+metadata:
+  name: cluster
+spec:
+  servingCerts:
+    namedCertificates:
+    - names:
+      - {{ .Values.cluster.apiServer.name }}
+      servingCertificate:
+        name: {{ .Values.cluster.apiServer.tlsSecret }}
+
+---
+apiVersion: operator.openshift.io/v1
```

Comment:
We should mention some of the permissions required here if we don't already. Looks like this is going to touch some admin-level components here.